ui/v2.5/src/components/Shared/ExternalLinksButton.tsx

```diff
@@ -47,3 +49,37 @@ export const ExternalLinksButton: React.FC<{
     </Dropdown>
   );
 };
+
+export const ExternalLinkButtons: React.FC<{ urls: string[] | undefined }> = ({
+  urls,
+}) => {
+  const urlSpecs = useMemo(() => {
+    if (!urls?.length) {
+      return [];
+    }
+
+    const twitter = urls.filter((u) =>
+      u.match(/https?:\/\/(?:www\.)?twitter.com\//)
```

Comment:
Might as well support the new domain while we're at it :)

```suggestion
      u.match(/https?:\/\/(?:www\.)?(?:twitter|x).com\/)
```